encord/objects/ontology_structure.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from enum import IntEnum
+from typing import List, TypeVar
+from uuid import uuid4
+
+from encord.objects.classification import Classification
+from encord.objects.common import Shape
+from encord.objects.ontology_object import Object
+
+AVAILABLE_COLORS = (
+    "#D33115",
+    "#E27300",
+    "#16406C",
+    "#FE9200",
+    "#FCDC00",
+    "#DBDF00",
+    "#A4DD00",
+    "#68CCCA",
+    "#73D8FF",
+    "#AEA1FF",
+    "#FCC400",
+    "#B0BC00",
+    "#68BC00",
+    "#16A5A5",
+    "#009CE0",
+    "#7B64FF",
+    "#FA28FF",
+    "#B3B3B3",
+    "#9F0500",
+    "#C45100",
+    "#FB9E00",
+    "#808900",
+    "#194D33",
+    "#0C797D",
+    "#0062B1",
+    "#653294",
+    "#AB149E",
+)
+
+
+@dataclass
+class OntologyStructure:
```

Comment:
I think this class would really benefit from a usage example. For example, check the `ontology` property on the `Project` class in `project.py`. Ideally as part of the doc strings we would explain how to create an `OntologyStructure` with some objects and attributes etc. Just to give the client an idea of how to use it.

Reply:
That way we could also avoid adding an extra page into the docs!

Reply:
https://python.docs.encord.com/api.html#encord.project.Project.ontology reference to how these examples look like in our docs. Please don't forget to update the docs when you do this. Instructions on how to do so are in `docs/README.md`